tools/packaging/kata-deploy/helm-chart/Makefile

```diff
@@ -0,0 +1,29 @@
+
+VERSION_FILE := ../../../../VERSION
+
+release:
+	sed -i 's/appVersion: .*/appVersion: $(shell cat $(VERSION_FILE))/g' kata-deploy/Chart.yaml
+all: package
+
+package: helm release
```

Comment:
I was expecting a "helm install" somewhere in our CI webhooks/tests, that would consume this package.

So do you mind expanding a little bit on the plan? What is next? Who would be calling this target and who would be consuming the `kata-deploy-0.1.0.tgz` helm package ?

Your commit message says "get all the base and overlays", should we try to replace it now within this PR?

Reply:
Yes that is the idea to use the helm chart for our CI and for our releases for users to have a pinned version to install from. I did not want to overload this PR just kicking of the discussion and see where and how to extend this.